cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -137,15 +138,26 @@ public void execute() {
             } else if (Files.isDirectory(sourceRootPath)) { // If the source is a package from a project
                 // Checks if the source is a package and if its inside a project (with a .ballerina folder)
                 if (Files.isDirectory(resolvedFullPath) && !RepoUtils.hasProjectRepo(sourceRootPath)) {
-                    outStream.println("error: do you mean to build the ballerina package as a project? If so run" +
-                                              " ballerina init to make it a project with a .ballerina directory");
-                    return;
+                    throw LauncherUtils.createLauncherException("did you mean to build the package ? If so build " +
+                                                                        "from the project folder");
+                }
+                // If we are trying to run a bal file inside a package from a project directory an error is thrown.
+                // To differentiate between top level bals and bals inside packages we need to check if the parent of
+                // the sourcePath given is null. If it is null then its a top level bal else its a bal inside a package
+                Path parentPath = sourcePath.getParent();
+                if (Files.isRegularFile(resolvedFullPath) && sourcePath.toString().endsWith(BLANG_SRC_FILE_SUFFIX) &&
+                        parentPath != null) {
+                    Path fileName = parentPath.getFileName();
+                    String srcPkgName = fileName != null ? fileName.toString() : "";
+                    throw LauncherUtils.createLauncherException("you are trying to build a ballerina file inside a " +
+                                                                        "package within a project. Try running " +
+                                                                        "'ballerina build <package-name>'");
                 }
             } else {
                 // Invalid source file provided
-                outStream.println("error: invalid Ballerina source path, it should either be a directory or a" +
-                                          "file  with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension");
-                return;
+                throw LauncherUtils.createLauncherException("invalid Ballerina source path, it should either be a " +
```

Comment:
B is in caps, but the rest of the error messages its not the case. Shall we be consistent?

Reply:
Made the error message lower case with f0356fb